mmeds/config.py

```diff
@@ -684,6 +684,12 @@
     col_type = COLUMN_TYPES_SPECIMEN[COL_TO_TABLE[col]][col]
     SAMPLE_ID_COLUMNS[col] = col_type
 
+# Demultiplexing Qiime Defaults
+QIIME_SAMPLE_ID_CATS = ('#SampleID', '#q2:types')
+QIIME_FORWARD_BARCODE_CATS = ('BarcodeSequence', 'categorical')
+QIIME_REVERSE_BARCODE_CATS = ('BarcodeSequenceR', 'categorical')
+QIIME_FILENAME_TEMPLATE = '{}_S1_L001_R{}_001.fastq.gz'
```

Comment:
The filename is not QIIME-related. Each sequencing center might have different ways of generating the files and naming them. Even if all were following this convention, you might have multiple lanes, etc. So this is most likely to fail at some point in the future.

Reply:
So unless we use this template in a different function or with a library other than `pheniqs`, this will always work. This is because we are only looking at the pheniqs output, which we control the names of with the other script, `make_pheniqs_config.py` (which is next on my list to update). The lane number for example is just because qiime can't read these files if they don't have the standard naming convention, but it will always be 'L001' if it is from pheniqs.